x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/downsample_field.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiTextColor } from '@elastic/eui';
+import { UnitField } from './unit_field';
+import { fixedIntervalUnits } from '../../../constants';
+import { UseField } from '../../../form';
+import { NumericField } from '../../../../../../shared_imports';
+import { ToggleFieldWithDescribedFormRow } from '../../described_form_row';
+// import { LearnMoreLink } from '../../learn_more_link';
+import { i18nTexts } from '../../../i18n_texts';
+import { PhaseWithDownsample } from '../../../../../../../common/types';
+
+interface Props {
+  phase: PhaseWithDownsample;
+}
+
+export const DownsampleField: React.FunctionComponent<Props> = ({ phase }) => {
+  // const { docLinks } = useKibana().services;
+
+  const downsampleEnabledPath = `_meta.${phase}.downsample.enabled`;
+  const downsampleIntervalSizePath = `_meta.${phase}.downsample.fixedIntervalSize`;
+  const downsampleIntervalUnitsPath = `_meta.${phase}.downsample.fixedIntervalUnits`;
+
+  return (
+    <ToggleFieldWithDescribedFormRow
+      title={
+        <h3>
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.downsampleTitle"
+            defaultMessage="Downsample"
+          />
+        </h3>
+      }
+      description={
+        <EuiTextColor color="subdued">
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.downsampleDescription"
+            defaultMessage="Roll up documents within a fixed interval to a single summary document. Reduces the index footprint by storing time series data at reduced granularity."
+          />{' '}
+          {/* TODO: add when available*/}
+          {/* <LearnMoreLink docPath={docLinks.links.elasticsearch.ilmDownsample} /> */}
```

Comment:
Is this documentation planned, or already in progress? I'm wondering if it makes sense to remove this code until it is ready.

Reply:
I believe this is planned: https://github.com/elastic/kibana/issues/130437#issuecomment-1226598533 cc @debadair